packages/cozy-client/src/CozyClient.js

```diff
@@ -1531,7 +1531,11 @@ instantiation of the client.`
    * Returns whether the client has been revoked on the server
    */
   async checkForRevocation() {
-    return this.stackClient.checkForRevocation()
+    if (this.stackClient instanceof OAuthClient) {
+      return this.stackClient.checkForRevocation()
+    } else {
+      throw 'checkForRevocation is only implemented for OAutClient'
+    }
```

Comment:
You have removed the method `checkForRevocation` in OAuthClient. Either you remove this API here, or, and I would prefer, let the checkForRevocation in OAuthClient

Reply:
My bad. It stills there